cmd/gardener-extension-provider-openstack/app/app.go

```diff
@@ -237,6 +237,16 @@ func NewControllerManagerCommand(ctx context.Context) *cobra.Command {
 				return fmt.Errorf("could not add ready check for webhook server to manager: %w", err)
 			}
 
+			// TODO (georgibaltiev): Remove after the release of version 1.51.0
```

Comment:
```suggestion
			// TODO (georgibaltiev): Remove after the release of version 1.50.0
```

The latest release is v1.47.0. The PR will be included in v1.48, v1.49 and v1.50. So after the release of v1.50 we could drop it, right? It would be present for 3 releases.

Reply:
Yes, I miscounted. Thanks for noticing!